### DISCUSSION TITLE
Discussion on HA forum?

### DISCUSSION THREAD

**Discussion by HenrikHills** [humbertogontijo/homeassistant-roborock#112] [Q&A]
Does this integration have a thread on HA forum or any discussion-board except here on Github? Maybe add that to the readme so people know of ongoing issues etc?

**Comment by wgumaa** [comment]
Look here: 
https://community.home-assistant.io/t/roborock-s7maxv-ha-integration/431453/54

**Reply by HenrikHills** [reply]
cheers! I guess I was looking for a more "generic roborock" thread

**Reply by wgumaa** [reply]
There are a lot for the xiaomi vaccum. This is the first roborock (unoffical) integration thats why.

**Reply by wgumaa** [reply]
the xiaomi also works with roborock
